sls-gallery.yml

```diff
@@ -0,0 +1,118 @@
+# Installation notes:
+# - Set up OME RHEL 7 machine.
+# - Set host variables:
+#   tls_certificate_bundle, tls_certificate_key,
+#   database_password, server_password
+# - Run playbook.
+# - systemctl stop omero-{server,web}
+# - Migrate binary repository to /OMERO/ with rsync.
+# - Migrate database with pg_dump, pg_restore.
+# - Upgrade database.
+# - bin/omero db password for root user.
+# - systemctl start omero-{server,web}
+
+- hosts: omero-sls-gallery
+  roles:
+    - role: ome.lvm-partition
+      lvm_vgname: VolGroup00
+      lvm_lvname: 'var_lib_psql'
+      lvm_lvmount: '/var/lib/pgsql'
+      lvm_lvsize: 8G
+      lvm_lvfilesystem: xfs
+      lvm_shrink: False
+
+    - role: ome.postgresql
+      postgresql_databases:
+        - name: omero
+      postgresql_users:
+        - user: omero
+          password: "{{ database_password }}"
+          databases: [omero]
+      postgresql_version: "10"
+
+    - role: ome.lvm-partition
+      lvm_vgname: VolGroup00
+      lvm_lvname: 'omero'
+      lvm_lvmount: '/OMERO'
+      lvm_lvsize: 80G
+      lvm_lvfilesystem: xfs
+      lvm_shrink: False
+
+    - role: ome.omero-server
+      ice_python_wheel: https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
+      omero_server_systemd_start: False
+      omero_server_release: latest
+      omero_server_dbpassword: "{{ database_password }}"
+      omero_server_rootpassword: "{{ server_password }}"
+      omero_server_config_set:
+        omero.client.ui.menu.dropdown.colleagues.enabled: False
+        omero.client.ui.menu.dropdown.everyone.label: "All images"
+        omero.client.ui.menu.dropdown.leaders.label: "Gallery:"
+        omero.db.poolsize: 50
+        omero.jvmcfg.percent.blitz: 50
+        omero.jvmcfg.percent.indexer: 20
+        omero.jvmcfg.percent.pixeldata: 30
+        omero.ldap.config: True
+        omero.ldap.base: "{{ omero_server_ldap_base }}"
+        omero.ldap.username: cn=ldapuser,ou=system,o=dundee
```

Comment:
Move all the LDAP config to the private config? In general anything that couldn't be run by an external user shouldn't be here

Reply:
Where do these get defined for our other servers? (Trying to see what the variable names are so I can reuse them but not seeing much LDAP config.)

Reply:
Looks like should do likewise for mail server variables.